### PR TITLE
Mute ReplicationTrackerRetentionLeaseTests#testAddOrRenewRetentionLease 6.x

### PR DESCRIPTION
**Pull request by cbuescher**

### CODE REVIEW

**Review by jasontedor — CHANGES_REQUESTED** [review]
I left a question? A reproduce with line would help so I can fix this.

File: server/src/test/java/org/elasticsearch/index/seqno/ReplicationTrackerRetentionLeaseTests.java [hunk 1]
```diff
@@ -50,6 +50,7 @@
 
 public class ReplicationTrackerRetentionLeaseTests extends ReplicationTrackerTestCase {
 
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/38245")
```
[the remark this answers is not in the document]
Comment (cbuescher) [reply server/src/test/java/org/elasticsearch/index/seqno/ReplicationTrackerRetentionLeaseTests.java]: The reproduce line is at https://github.com/elastic/elasticsearch/issues/38245#issuecomment-460042530.
Yes, I see now this is a different method in the same test suite. Will add the error to the issue, it looks quite similar but we can open a separate issue if this is necessary.

[the remark this answers is not in the document]
Comment (jasontedor) [reply server/src/test/java/org/elasticsearch/index/seqno/ReplicationTrackerRetentionLeaseTests.java]: I opened #38280.

**Review by jasontedor — APPROVED** [review]
LGTM.

**Pull request merged**